setup.py

```diff
@@ -226,14 +246,15 @@ def build_pkg(setup_file, author, author_email, description, url):
     Returns:
 
     """
-    pkg_dir = os.path.dirname(os.path.realpath(setup_file))
+    print(f"Build wheel from {setup_file}")
+    pkg_dir = setup_file.parent
     # package description
     pkg_pip_name = "power-grid-model"
     pkg_name = pkg_pip_name.replace("-", "_")
-    with open(os.path.join(pkg_dir, "README.md")) as f:
+    with open(pkg_dir / "README.md") as f:
         long_description = f.read()
         long_description = convert_long_description(long_description)
-    with open(os.path.join(pkg_dir, "requirements.txt")) as f:
+    with open(pkg_dir / "requirements.txt") as f:
         required = f.read().splitlines()
         required = [x for x in required if "#" not in x]
```

Comment:
It is possible to add comments to your pip package, e.g.
```pip
fancypackage==1.2.3  # Use version pin until bug #42 is solved
```

So I would recommend to use `if lstrip(x) and lstrip(x)[0] != "#"`

Reply:
What should you import for `strip`?

Reply:
Sorry, I meant `x.strip()`